### DISCUSSION TITLE
V needs small C outputs

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#7646] [Ideas]
I saw a language called [nelua](nelua.io) and it generates surprisingly small C files( ~60 LOC for hello world). V also needs to output small C files , it would be a lot faster!

**Comment by Delta456** [comment]
Try using `-x64` backend or `-usecache` then see.

**Reply by ghost** [reply]
I know I can use them, I don't care about speed that much, but it also would be more readable. Also ```-x64``` isn't mature yet.

**Reply by Delta456** [reply]
It's a progress and soon will be focused more.

**Comment by medvednikov** [comment]
Yes, this is important to have, it's on the roadmap.

Will be done by 0.3.
